### ISSUE TITLE
Data Story: California State Parks Data

### ISSUE THREAD

**Issue by feep** [qri-io/data-stories-scripts#6]
Two stories:

### Generate and use dataset
- [x] Generate csv from https://www.parks.ca.gov/ParkIndex
- [x] Generate sample file, `.kmx` location file to be used with gps software

### Improve dataset
- [ ] `meta.json`
- [ ] `structure.json`
- [ ] `transform.star` to do what the python script did in the first story

**Comment by feep** [comment]
Two blockers:

- [ ] [Can’t figure out how to allow `None` in `structure.json`](https://discordapp.com/channels/497052800778502152/580806684088336395/695060285312073819)
- [ ] [Totally broken qri, can’t use desktop or checkout the correct dataset](https://discordapp.com/channels/497052800778502152/580806684088336395/694933103898001429)

**Comment by feep** [comment]
We can cut the validation bit, and I can `publish` the dataset to a new user if you want the story to go asap.

**Comment by feep** [comment]
`structure.json` bit deferred to a separate story, so we can post this.

Especially since @Arqu is [working on jsonschema](https://discordapp.com/channels/497052800778502152/580806684088336395/702519607596154950).

https://github.com/qri-io/jsonschema/pull/63 and future PRs.

**Comment by Arqu** [comment]
Oh, just to add to this, think you should definitely proceed with this and "unblock" yourself for the time being by doing [what Holger did](https://discordapp.com/channels/497052800778502152/580806684088336395/702520685393739878) and you can refer to [this](https://stackoverflow.com/questions/16241333/specify-a-value-can-be-a-string-or-null-with-json-schema) for a bit more input. 

In a nutshell, for the problematic onces you should do an `anyOf` with your current validation rules + `type [string, null]` to account for the None/Null values. A bit of fiddling should get you to a spot that works. It's not pretty, but should get you over the hump. The jsonschema work will take a while to land, especially the part with nullables.

**Comment by feep** [comment]
Right, I got the working JSON from Holger.

=]

Thanks, Holger.

Anyway still deferring `structure.json` for other reasons. 

Will write later. Maybe with new "AllowNullable" Maybe without.
